Web.md

```diff
@@ -60,15 +60,18 @@ as described in the [`WebAssembly.Module` constructor](#webassemblymodule-constr
 On success, the `Promise` is [fulfilled](http://tc39.github.io/ecma262/#sec-fulfillpromise)
 with the resulting `WebAssembly.Module` object. On failure, the `Promise` is
 [rejected](http://tc39.github.io/ecma262/#sec-rejectpromise) with a
-`WebAssembly.CompileError`.
+`WebAssembly.CompileError` or `TypeError`, depending on the type of failure.
```

Comment:
Nit: should we say "depending on the type of failure, as it will be clarified further below"

... or something to that extent?